### DISCUSSION TITLE
I am lost with liabilities

### DISCUSSION THREAD

**Discussion by pedrom34** [firefly-iii/firefly-iii#7328] [Firefly III]
Hello everyone,

I have two current home mortgages, created before firefly-iii 5.8. So I have two liabilities in firefly-iii. The money was spent to buy my house (I never had the money on any of my accounts), I pay back my bank every month.
The accounts are both negative and I have imported the schedule of payments into firefly-iii, so every month I am getting closer to 0. This is fine with me. I understood that the behaviour has changed, but I'm not getting how it impacts me.

The thing is, I just signed up for a new personal loan. The money has arrived on my account, and I have a new schedule, and no interest (so this is kinda new for me). I can't figure out the best way to record this data in Firefly-iii.

My question are:
- do I have to change the old liabilities?
- can you help me with the best way to record my new loan? Do I have to create a liability, or just record the amount and import the schedule?

Thanks!

**Comment by JC5** [comment]
You don't have to change anything in your old liabilities, since this will not affect you. The new loan can be created as a liability and set the you can just set the amount when you create it.
